webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -0,0 +1,251 @@
+                            }))
+                            handleAnwsers.push(<div key={choices}>{choices}</div>)
+                        }
+                        if (answer.type.includes("text")) {
+                            handleAnwsers.push(<div>
+                                <div key={answer.value} data-tip={answer.value}><p>{answer.value}</p><ReactTooltip
+                                    className="Tooltip"
+                                />
+                                </div>
+                                
+                            </div>)
+                        }
+                    }
+                })
+
+                // extract only the reviewers name
+                val.reviewers.map(review => {
+                    review ? handleReviews.push(review.reviewer_name) : handleReviews.push("")
+                })
+                // envoke and store new columns for Reviews and Answers
+                let addAnwsers = addAnswersCol(val, handleAnwsers);
+                let addReviews = addReviewCol(val, handleReviews);
+
+                // insert anwsers values as columns
+                if (handleAnwsers.length) {
+                    addAnwsers.map((answer, index) => {
+                        let num = (index) + (1);
+                        let key = "Answer" + num;
+                        val[key] = answer
+                    })
+                }
+                // insert new reviews values as columns
+                addReviews.map((review, index) => {
+                    let num = (index) + (1);
+                    let key = "Review" + num;
+                    val[key] = review
+                })
+                // delete original review and answer rows as they don't need to be displayed with all their data
+                handleAnwsers = [];
+                handleReviews = [];
+                delete val.answers;
+                delete val.reviewers;
+            })
+
+            this.setState({
+                responseTable: response,
+                addBtn: { background: "hsl(211deg 100% 50%)" }
+            }
+            )
+
+        })
+    }
+
+
+    generateCol() {
+        let colFormat = [];
+        let colStyle = {
+            'maxWidth': '300',
+            'width': '250',
+            'whiteSpace': 'normal',
+            'maxHeight': '200px',
+            'overflow': 'scroll',
+            'marginBottom': '10px'
+        }
+
+        if (this.state.responseTable) {
+            function widthCalc(colItem) {
+                if (colItem.includes('Answer')) {
+                    return 200
+                }
+                if (colItem.includes('Review')) {
+                    return 180
+                }
+                else {
+                    return 100
+                }
+            }
+
+            let col = Object.keys(this.state.responseTable[0]);
+            col.map(val => {
```

Comment:
The "correct" use of map here would be:
`colFormat = col.map(val => ({id: val, Header: val, accessor: val, style: colStyle, width: widthCalc(val)}))`